doc/themes/nilearn/layout.html

```diff
@@ -105,10 +105,10 @@
         for(i in sections){
             if(sections[i] > pos){
 		break;
-            };
-	    if($('a.internal[href$="' + i + '"]').is(':visible')){
+            }
+            if($('a.internal[href$="' + i + '"]').is(':visible')){
```

Comment:
This seems unrelated.
Pelase try to keep PRs specific.

Reply:
I don't know why the HTML & JS files changed. I used Sphinx to build them for testing. I didn't change their code manually. Seems they were inadvertently auto-added and committed by Git.